source/wp-content/themes/wporg-developer-2023/src/search-filters/index.php

```diff
@@ -62,8 +62,9 @@ function render( $attributes, $content, $block ) {
 
 	$wrapper_attributes = get_block_wrapper_attributes();
 	return sprintf(
-		'<div %1$s>%2$s</div>',
+		'<div %1$s>%2$s <button>%3$s</button></div>',
```

Comment:
Not required, but it could be more clear to explicitly set the type:
```suggestion
		'<div %1$s>%2$s <button type="submit">%3$s</button></div>',
```